src/services/staff/staff-schemas.ts

```diff
@@ -21,7 +21,7 @@ export const StaffAttendanceRequestSchema = z.object({
 export const ScanAttendeeRequestSchema = z
     .object({
         eventId: EventIdSchema,
-        attendeeJWT: z.string().openapi({ description: "The scanned QR code token", example: "a35FG==" }),
+        qrId: z.string().openapi({ description: "The scanned QR code token", example: "ns6Shwu2" }),
```

Comment:
this is a breaking change. If we are changing it, we should have a better name like `attendeeQRCode`